docs/content/concepts/object-ownership/wrapped.mdx

```diff
@@ -70,9 +70,9 @@ public fun new(scarcity: u8, style: u8, ctx: &mut TxContext): Object {
 
 ```
 
-You can also enable a swap/trade between your object and others' objects. For example, define a function that takes two objects from two addresses and swaps their ownership. But this doesn't work in Sui. Only object owners can send a transaction to mutate the object. So one person cannot send a transaction that would swap their own object with someone else's object.
+You can also enable a swap or trade between your object and others' objects. For example, define a function that takes two objects from two addresses and swaps their ownership. This doesn't work in Sui. Only object owners can send a transaction to mutate the object. One person cannot send a transaction that would swap their own object with someone else's object.
```

Comment:
This brings up an interesting point...numbers (as in digits) probably shouldn't begin a sentence.